content/docs/command-reference/plots/index.md

```diff
@@ -99,14 +99,17 @@ can specify only the base name e.g. `--template scatter`.
 
 ### Custom templates
 
-Plot template files are just JSON specifications with predefined DVC anchors
-that help DVC to inject user's data properly. You can create a custom template
-from scratch or modify an existing one from `.dvc/plots/`. Custom templates can
-be added to the template directory.
+Plot template files are just
+[Vega specification](https://vega.github.io/vega/docs/specification/) files that
+use predefined DVC anchors as placeholders for DVC to inject the plot values.
+You can create a custom template from scratch, or modify an existing one from
+`.dvc/plots/`.
 
-All JSON files given to `dvc plots show` and `dvc plots diff` as input are
-combined together into a single data array for the injection to a template file.
-There are two important fields that DVC adds to the plot data:
+> Note that custom templates can be added to the template directory safely.
```

Comment:
not sure where that note comes from?

Reply:
It was part of the paragraph above before, I separated it this way. Just made it into an emoji note instead. See 4729c0d2. Thanks